app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -58,25 +63,27 @@ public static int getSize() {
 
     public Mood(){
         username = null;
-        emotionalState = null;
-        description = null;
+        emotionalState = "";
+        description = "";
```

Comment:
Emotional State and description should never be a null field since we always fill them out one way or another. If we want to change this it'll require a hefty amount of "check if" code.